ocs_ci/ocs/ui/page_objects/encryption_module.py

```
@@ -92,6 +87,64 @@ def _get_encryption_summary(self, context_key):
+        """
+        row = None
+        for row_xpath in (
+            "./parent::*/parent::*",
+            "./ancestor::*[contains(@class, 'l-flex') and not(contains(@class, '__item'))][1]",
+            "./ancestor::*[contains(@class, 'flex') and not(contains(@class, '__item'))][1]",
+        ):
+            try:
+                row = header_element.find_element(By.XPATH, row_xpath)
+                break
+            except Exception:
+                continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)